consensus/clique/src/main/java/tech/pegasys/pantheon/consensus/clique/jsonrpc/methods/Propose.java

```diff
@@ -38,10 +40,12 @@ public String getName() {
   public JsonRpcResponse response(final JsonRpcRequest request) {
     final Address address = parameters.required(request.getParams(), 0, Address.class);
     final Boolean auth = parameters.required(request.getParams(), 1, Boolean.class);
-    if (auth) {
-      proposer.auth(address);
-    } else {
-      proposer.drop(address);
+    if (!address.equals(NO_VOTE_ADDRESS)) {
+      if (auth) {
+        proposer.auth(address);
+      } else {
+        proposer.drop(address);
+      }
```

Comment:
this should raise an error if the vote can't be cast.

Reply:
done